apps/skilavottord/web/screens/AccessControl/AccessControl.tsx

```diff
@@ -0,0 +1,334 @@
+import React, { FC, useContext, useState } from 'react'
+import { useMutation, useQuery } from '@apollo/client'
+import gql from 'graphql-tag'
+import startCase from 'lodash/startCase'
+import NextLink from 'next/link'
+
+import {
+  Box,
+  Breadcrumbs,
+  Button,
+  Stack,
+  Text,
+  Table as T,
+  GridColumn,
+  GridRow,
+  SkeletonLoader,
+} from '@island.is/island-ui/core'
+import { PartnerPageLayout } from '@island.is/skilavottord-web/components/Layouts'
+import { useI18n } from '@island.is/skilavottord-web/i18n'
+import Sidenav from '@island.is/skilavottord-web/components/Sidenav/Sidenav'
+import {
+  hasPermission,
+  isDeveloper,
+} from '@island.is/skilavottord-web/auth/utils'
+import { UserContext } from '@island.is/skilavottord-web/context'
+import { NotFound } from '@island.is/skilavottord-web/components'
+import { filterInternalPartners } from '@island.is/skilavottord-web/utils'
+import { useLinkResolver } from '@island.is/web/hooks/useLinkResolver'
```

Comment:
This is a whole another application (not a library) ... we do not want to pull in that bundle